build.gradle

```diff
@@ -25,7 +25,6 @@ plugins {
     id 'org.sonarqube' version '2.8'
     id "org.flywaydb.flyway" version "6.2.0"
     id 'maven-publish'
-    id 'maven'
```

Comment:
any reason for removing this?

Reply:
Plugin [id: 'maven'] was not found in any of the following sources:
The maven plugin has been removed. You should use the maven-publish plugin instead.